README.md

```diff
@@ -10,6 +10,7 @@
 # Novu Java SDK
 
 [![License](https://poser.pugx.org/unicodeveloper/novu/license.svg)](LICENSE.md)
+[![SDK Version](https://img.shields.io/badge/SDK-v1.4.0-blue.svg)](https://github.com/novuhq/novu-java/releases/tag/v1.4.0)
```

Comment:
Would the version change dynamically?

Reply:
It would have to be updated when there's a new release

Reply:
A dynamic badge would be great

Reply:
I have made it dynamic @mayorJAY